utils/appliance/endpoints/__init__.py

```diff
@@ -1,15 +0,0 @@
-class Endpoint(object):
```

Comment:
I kinda liked having an inherited endpoint object that we could use for testing if an object is an Endpoint object. I think it could be useful........thoughts? @mfalesni @RonnyPfannschmidt @jkrocil